command-expansion-server/jestGlobalSetup.js

```diff
@@ -11,57 +11,59 @@ process.env.MERLIN_GATEWAY_URL = 'http://localhost:9000';
 process.env.MERLIN_GRAPHQL_URL = 'http://localhost:8080/v1/graphql';
 
 export default async () => {
-  const ssoFilePath = `${os.tmpdir()}/.aerie/sso.txt`;
-  try {
-    const ssoToken = await fs.promises.readFile(ssoFilePath, 'utf8');
-    const res = await fetch(process.env.MERLIN_GATEWAY_URL, {
-      headers: { 'x-auth-sso-token': ssoToken },
-    });
-    if (!res.ok) {
-      throw new Error(`${res.status} ${res.statusText}`);
-    }
-    process.env.SSO_TOKEN = ssoToken;
-  } catch (e) {
-    await fs.promises.rm(ssoFilePath, { force: true });
-    const mutableStdout = new Writable({
-      write(chunk, encoding, callback) {
-        if (!this.muted) process.stdout.write(chunk, encoding);
-        callback();
-      },
-    });
-    const readlineInterface = readline.createInterface({
-      input: process.stdin,
-      output: mutableStdout,
-      terminal: true,
-    });
+  // Removing this because it appears we don't need it with auth: none, keeping it here in case we need it in the future though
```

Comment:
Just a thought: My preference would be to use the Git history to retrieve this if it is indeed needed in the future. My thinking here is that as we get further away from the time when we first commented out code, we slowly lose the context of why  we did so. The Git history with commit messages is a good way to document those decisions.

Reply:
I removed it - authenticating for tests isn't something I think we generally want, so I can't imagine this code being worth keeping around

Reply:
Myself not being very familiar with this particular test set up, is there a way to factor out the login code to say a separate file that can be kept along side and selectively used?

Reply:
> I removed it 
Okay 👍